br/pkg/streamhelper/advancer.go

```diff
@@ -413,10 +413,12 @@ func (c *CheckpointAdvancer) onTaskEvent(ctx context.Context, e TaskEvent) error
 	defer c.taskMu.Unlock()
 	switch e.Type {
 	case EventAdd:
+		utils.SetLogBackupTaskExist(true)
```

Comment:
Will there be multiple tasks? If so, it's better to report tasks counter to the telemetry server.